src/test/scala/org/ergoplatform/utils/Stubs.scala

```diff
@@ -134,12 +135,13 @@ trait Stubs extends ErgoGenerators with ErgoTestHelpers with ChainGenerator with
 
     private implicit val addressEncoder: ErgoAddressEncoder = new ErgoAddressEncoder(settings.chainSettings.addressPrefix)
     private val prover = new ErgoProvingInterpreter(seed, 2)
-    private val trackedAddresses: mutable.Buffer[ErgoAddress] =
-      mutable.Buffer(prover.dlogPubkeys: _ *).map(P2PKAddress.apply)
-
+    private val trackedAddresses: Seq[P2PKAddress] = prover.dlogPubkeys.map(P2PKAddress.apply)
 
     def receive: Receive = {
 
+      case ReadPublicKeys(from, until) =>
```

Comment:
It is not used anywhere

Reply:
@catena2w, it is required for tests as wallet route now read public keys from wallet, so we need stub to mock wallet actor response.
https://github.com/ergoplatform/ergo/blob/1db706f65805cac501f0464fe8674670e83a46ef/src/main/scala/org/ergoplatform/api/WalletApiRoute.scala#L144